Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -1094,6 +1097,8 @@
     <RegEx find="stomačn" replaceWith="trbušn" />
     <RegEx find="Stomačn" replaceWith="Trbušn" />
     <RegEx find="\b([sS])trj?el([aeiouc])" replaceWith="$1trijel$2" />
+    <RegEx find="sugeriše([mšt])" replaceWith="predlaže$2" />
+    <RegEx find="Sugeriše([mšt])" replaceWith="Predlaže$2" />
```

Comment:
I meant to ask you, could Word List Validator discover that this is an error, or it's entirely up to us?